.github/workflows/ci.yml

```
@@ -48,12 +48,12 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v4
```

actionlint (1.7.10)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)